File: Java/leetcode75/src/test/java/org/phoenix/leetcode/challenges/Problem22_DetermineIfTwoStringsAreCloseTest.java
package org.phoenix.leetcode.challenges;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertTrue;

class Problem22_DetermineIfTwoStringsAreCloseTest {

    private final Problem22_DetermineIfTwoStringsAreClose test = new Problem22_DetermineIfTwoStringsAreClose();

    @Test
    void closeStrings() {
        String word1 = "cabbba", word2 = "abbccc";
        assertTrue(test.closeStrings(word1, word2));
    }
}

File: JavaScript/leetcode75/src/test/Problem22_DetermineIfTwoStringsAreClose.test.js
const closeStrings = require("../main/Problem22_DetermineIfTwoStringsAreClose");

test("should return whether it is possible or not", () => {
  const word1 = "cabbba",
    word2 = "abbccc";
  expect(closeStrings(word1, word2)).toBe(true);
});
